pytentiostat/tester.py

```diff
@@ -85,12 +102,25 @@ def read_write(
         Conversion factor to correct the current and voltage readings
     sr: float
         Shunt resistor used to correct the current reading
-    data: dict
+    config_data: dict
         Dictionary that contains the data read from the config file.
+    times: list
+        Contains the time measured during the experiment at each data point

```

Comment:
``The list of <something> measured during experiment at each point``.  time is ambiguous, does it mean wall-clock-time when the measurement was done or the measurement-time at the point?